targets/targets.json

```diff
@@ -5219,8 +5576,7 @@
         ],
         "device_name": "R7S72100",
         "bootloader_supported": true,
-        "mbed_rom_start": "0x18000000",
-        "mbed_rom_size": "0x800000",
+        "mbed_ram_size": "0xA00000",
```

Comment:
Does this device have ROM or flash? Why not have an `mbed_ram_start` to describe the RAM start address?